src/screens/discover/components/NetworkSwitcher.tsx

```diff
@@ -0,0 +1,822 @@
+                marginTop: 68 - 75,
+              }}
+            >
+              <LinearGradient
+                colors={['#268FFF1F', '#268FFF14']}
+                angle={135}
+                useAngle
+                style={{
+                  height: 36,
+                  width: 36,
+                  borderRadius: 10,
+                  borderWidth: 1,
+                  borderColor: '#268FFF0D',
+                  backgroundColor: '#268FFF14',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
+              >
+                <Text weight="heavy" size="17pt" color={{ custom: blue }}>
+                  􀍱
+                </Text>
+              </LinearGradient>
+              <View style={{ gap: 10 }}>
+                <Text weight="heavy" size="15pt" color="labelSecondary">
+                  {i18n.t(t.customize_networks_banner.title)}
+                </Text>
+                <Text weight="semibold" size="13pt" color="labelQuaternary">
+                  {/* 
+                    is there a way to render a diferent component mid sentence?
+                    like i18n.t(t.customize_networks_banner.description, { Edit: <Text... /> })
+                  */}
+                  Tap the{' '}
+                  <Text weight="bold" size="13pt" color={{ custom: blue }}>
+                    Edit
+                  </Text>{' '}
+                  button below to set up
+                </Text>
+              </View>
+              <Pressable style={{ marginLeft: 'auto', height: '100%' }} onPress={dismissCustomizeNetworksBanner}>
+                <Text weight="heavy" size="13pt" color="labelQuaternary">
+                  􀆄
+                </Text>
+              </Pressable>
+            </View>
+          </AnimatedBlurView>
+        </MaskedView>
+      </Animated.View>
+    </DesignSystemProvider>
+  );
+}
+
+export function NetworkSelector({ onClose, onSelect, multiple }: { onClose: VoidFunction; onSelect: VoidFunction; multiple?: boolean }) {
+  const { isDarkMode } = useTheme();
+  const surfacePrimary = useBackgroundColor('surfacePrimary');
+  const backgroundColor = isDarkMode ? '#191A1C' : surfacePrimary;
+  const separatorSecondary = useForegroundColor('separatorSecondary');
+  const separatorTertiary = useForegroundColor('separatorTertiary');
+  const fill = useForegroundColor('fill');
+
+  const translationY = useSharedValue<number>(0);
+
+  const swipeToClose = Gesture.Pan()
+    .onChange(event => {
+      if (event.translationY < 0) return;
+      translationY.value = event.translationY;
+    })
+    .onFinalize(() => {
+      if (translationY.value > 120) runOnJS(onClose)();
+      else translationY.value = withSpring(0);
+    });
+
+  const [isEditing, setEditing] = useState(false);
+
+  const animatedStyles = useAnimatedStyle(() => ({
+    transform: [{ translateY: translationY.value }],
+  }));
+
+  const [selected, setSelected] = useState<ChainId[] | 'all'>([]);
```

Comment:
should this be stored in the zustand store?

Reply:
I actually think it makes sense to move a lot of the internal helpers (selection, editing, etc.) to the Zustand store but that's my personal preference